src/views/navigation/notificationsTab.js

```diff
@@ -39,6 +40,9 @@ type Props = {
 const NotificationsTab = (props: Props) => {
   const { count, data, isActive, match, currentUser } = props;
 
+  const isWideViewport =
+    window && window.innerWidth > MIN_WIDTH_TO_EXPAND_NAVIGATION;
```

Comment:
This will break SSR rehydration any logged-in page I think? We should do this with CSS media queries inside the Tooltip component.

Reply:
I thought about this before shipping, but I can't see it actually having an impact even on production. Are you seeing issues? If so, happy to revert, I know this is hacky anyways